src/master-playlist-controller.js

```diff
@@ -14,6 +14,13 @@ import Decrypter from './decrypter-worker';
 
 let Hls;
 
+// Default codec parameters if non were provided for video and/or audio
+const defaultCodecs = {
+  videoCodec: 'avc1',
+  videoObjectTypeIndicator: '.4d400d',
```

Comment:
I'm not sure there is a reason other than "This was a common configuration when we started writing this project..." which is a crappy comment :1st_place_medal: